tests/parm/regional_atmaq.nml.IN

```diff
@@ -316,6 +316,7 @@
     fnsnoa = ''
     fnsnoc = 'global_snoclim.1.875.grb'
     fnsotc = @[FNSOTC]
+    fnsocc = @[FNSOCC]
```

Comment:
I am curious, do we need to define fnsocc for regional tests? Also why regional_atmaq.nml.IN needs fnsocc, while the regional_stoch.nml.in only needs iopt_diag?

Reply:
@barlage I think we should use soil color data for regional tests as well. I added fnsocc setting to regional_stoch.nml.in. This is for very high resolution test (C3342). Can Sanath create this dataset?

Reply:
@HelinWei-NOAA There are no grids C3342 in the /scratch1/NCEPDEV/global/glopara/fix/orog/20220805. I am not sure how I can create without them.